modules/data_dictionary_widget/data_dictionary_widget.module

```diff
@@ -76,7 +76,7 @@ function data_dictionary_widget__data_dictionary_data_type_checker($context) {
 function data_dictionary_widget_form_alter(&$form, &$form_state, $form_id) {
   $formObject = $form_state->getFormObject();
 
```

Comment:
Add list of forms:   `$target_form_ids = ['node_data_edit_form', 'node_data_form'];`